conda/conda
# Release branch for 3.11

Didn't realize that I didn't have the latest code when I was on the plane working on #1273.  This creates a branch off of the latest release so the Windows upgrade changes are isolated and can be release independently.

This supersedes #1273.

## Steps to Test
* Before installing, create a new environment with pandas 0.16
* Open `python` console and type `import pandas` and `pandas.__version__` to verify the version is as expected
* Leave `python` console open
* In another terminal window, attempt to install `pandas=0.14*`
* When prompted, select `f` to force installation

At this point, you can repeats step 2 and you should receive an error.

To fix this, you have to close all running `python` consoles, then install `pandas` again.

Once you've reset, you can install this code and test -- it shouldn't allow you to force the installation.
